src/BuildAfterApp/BuildAfterApp.csproj

```diff
@@ -0,0 +1,79 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I prefer something like `BuildAfterSharedFramework.csproj` for this, `BuildAfterApp` is pretty vague (what app?)